guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -128,10 +128,7 @@
 
         "INFO_CONNECTION_SHARED" : "This connection is now shared.",
         "INFO_NO_FILE_TRANSFERS" : "No file transfers.",
-
-        "MESSAGE_DEFAULT"      : "",
-        "MESSAGE_USER_JOINED"  : "User {ARGS_1} has joined the connection.",
```

Comment:
Huh, these were just unused?

Reply:
Nope, but they are now. They were used by the message display directive that these changes replace.

Reply:
Ah, nevermind. I see where they were used now.